:sparkles: Add Saliva Kit to SAMPLE_PROCUREMENT_ENUM for Letra

r3m0chop opened the pull request:
✨ Add new Saliva Kit for sample procurement enum,

Note that `SAMPLE_PROCUREMENT.SALIVA_KIT` had already been added to the ingest lib (as at https://github.com/kids-first/kf-lib-data-ingest/pull/657), hence the use of "_Saliva Kit_" as the general variation to the following:
*  _"Saliva Collection - Proband"_
*  _"Saliva Collection - Maternal"_
*  _"Saliva Collection - Paternal"_

r3m0chop commented:
Looks like the `circleci` tests failed, and I am currently looking into these....

r3m0chop commented:
Got it!  My comments exceeded the PEP8 recommended length:
```
=================================== FAILURES ===================================
__________________________________ PEP8-check __________________________________
/home/circleci/repo/dataservice/api/biospecimen/schemas.py:31:80: E501 line too long (83 > 79 characters)
    # NOTE: SAMPLE_PROCUREMENT.SALIVA_KIT had already been added to the ingest lib,
                                                                               ^
/home/circleci/repo/dataservice/api/biospecimen/schemas.py:32:80: E501 line too long (83 > 79 characters)
    # hence the use of "Saliva Kit" as the general variation to "Saliva Collection"
                                                                               ^
```

r3m0chop commented:
Reduced the line lengths with a `git --amend` to the previous commit ; awaiting `circleci` to rerun tests....

Christina-J-Diaz commented:
Looks good!

znatty22 reviewed (approved):
👍

r3m0chop commented:
> Hey @r3m0chop could you please rebase on master before merging?

Hi, @znatty22 .  I've rebased to include your commits to `master` last night but now the `continuous-integration/jenkins/branch` is not building successfully, and I can't quite tell what's hanging it up.  Any ideas?  Thanks!

znatty22 reviewed (commented):
Hey @r3m0chop I just merged a PR so you'll have to rebase on master again. The Jenkins deployment went fine so I'm sure your PR will deploy fine too

r3m0chop commented:
Thanks, @znatty22 .  Rebased and updated to remove the previous trivial commit (hoping that the continuous integration tests go swimmingly and have no such need this time 🤞 )

znatty22 reviewed (approved):
👍, feel free to merge after the status checks pass!
